WalletWasabi.Documentation/FAQ.md

```diff
@@ -0,0 +1,178 @@
+
+# Install
+
+### How do I install Wasabi?
+
+Follow [this guide](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Documentation/Guides/InstallInstructions.md)
+
+If you use Windows there is a [video guide](https://youtu.be/D8U53PFEsVk).
+
+### Do I need to run Tor?
+
+All Wasabi network traffic goes via Tor by default - no need to set up Tor yourself. If you do already have Tor, and it is running, then Wasabi will try to use that first.  
+
+# Pre-Mix
+
+### My wallet can't send to Bech32 addresses - what wallets can I use instead? 
+
+Wasabi generates Bech32 addresses only. These addresses start with the characters bc1... Some wallets/exchanges don't yet support this type of address and my give an error message (e.g. "unknown bitcoin address"). The solution is to manage your funds with a wallet which does support Bech32. Of [all the wallets](https://en.bitcoin.it/wiki/Bech32_adoption) that support Bech32, only the following wallets have Coin Control;
+
+| Platform 						| Recommended 																					| Other										|
+| ---------						| -----------------------------------------------------	|------------------------	|
+| Mobile (Android)		| [Samourai](https://samouraiwallet.com/)								| [Electrum](https://play.google.com/store/apps/details?id=org.electrum.electrum)								|
+| Desktop 						| [Bitcoin Core](https://bitcoincore.org/en/download/)	| [Electrum](https://electrum.org)	|
+
+I am not aware of any wallets for iOS which support coin control. If you are aware of any other Wallets with coin control please make a pull request.
+
+Be careful. If you send all your coins from an old wallet to a new wallet (from the table above) in one transaction then you will merge all your coins which is bad for privacy. Instead, **import your seed into one of the wallets in the table.**
+
+# Mixing
+
+### What are the fees?
+
+You pay currently pay a fee of 0.003% * Anonymity set. If the coin anonymity set of a coin is 50 then you paid 0.003% * 50 (=0.15%). If you set the target anonymity set to 53 then wasabi will continue mixing until this is reached, so you may end up with an anonymity set of say 60, and you will pay 0.003% * 60 (=0.18%).
+
+### What is the Anonymity Set?
+
+The anonymity set is effectively the size of the group you are hiding in. 
+
+If 3 people take part in a CoinJoin (with equal size inputs) and there are 3 outputs then each of those output coins has an anonymity set of 3.
+
+```
+0.1 BTC (Alice)       0.1 BTC (Anon set 3)
+0.3 BTC (Bob) 	  ->  0.1 BTC (Anon set 3)
+0.4 BTC (Charlie)     0.1 BTC (Anon set 3)
+                      0.2 BTC (Change Coin Bob)
+                      0.3 BTC (Change Coin Charlie)
+```
+
+There is no way to know which of the anon set output coins are owned by which of the input owners.
+All an observer knows is that a specific anon set output coin is owned by one of the owners of one of the input Coins i.e. 3 people - hence an anonymity set of 3.
```

Comment:
```suggestion
All an observer knows is that a specific anon set output coin is owned by one of the owners of one of the input Coins i.e. 3 people - hence an anonymity set of 3.  
Your Wasabi software has limited information on what the anonymity set should be, so the anonymity set that the software presents you is just an estimation, not an accurate value. With Wasabi we are trying to do lower estimations, rather than upper ones.
```